recipes/hapnest/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: hapnest
+  version: "1"
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - julia >=1.6.4
```

Comment:
This does not look correct. Please look at the PR description for some instructions.

Reply:
Thank you, I fixed run_exports.